x-pack/plugins/security_solution/public/flyout/document_details/isolate_host/header.test.tsx

```diff
@@ -39,4 +43,19 @@ describe('<PanelHeader />', () => {
     expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toBeInTheDocument();
     expect(getByTestId(FLYOUT_HEADER_TITLE_TEST_ID)).toHaveTextContent('Release host');
   });
+
+  it.each(['isolateHost', 'unisolateHost'])(
```

Comment:
Does it make sense to add a small test to make sure of the non-existence of `beta` badge is it is not a Sentinal one alert.

Reply:
Absolutely

Reply:
Looks good. Thank you. Resolving this comment.